src/drivers/stm32/tone_alarm/tone_alarm.cpp

```diff
@@ -537,6 +540,8 @@ ToneAlarm::rest_duration(unsigned rest_length, unsigned dots)
 void
 ToneAlarm::start_note(unsigned note)
 {
+	if (circuit_breaker_enabled("CBRK_BUZZER", CBRK_BUZZER_KEY)) { return ; }
```

Comment:
Now it'll be interesting what it does with the space. My guess is this won't pass.